pkg/distro/fedora/images.go

```diff
@@ -161,39 +161,22 @@ func osCustomizations(
 		osc.YUMRepos = append(osc.YUMRepos, osbuild.NewYumReposStageOptions(filename, repos))
 	}
 
-	if oscapConfig := c.GetOpenSCAP(); oscapConfig != nil {
-		if t.rpmOstree {
-			panic("unexpected oscap options for ostree image type")
-		}
-
-		datastream := oscap.GetDatastream(oscapConfig.Datastream, t.arch.distro)
-
-		oscapStageOptions := osbuild.OscapConfig{
-			Datastream:  datastream,
-			ProfileID:   oscapConfig.ProfileID,
-			Compression: true,
-		}
-
-		osc.OpenSCAPTailorConfig = oscap.CreateTailoringStageOptions(
-			oscapConfig,
-			t.arch.distro,
-		)
-
-		if tailorConfig := osc.OpenSCAPTailorConfig; tailorConfig != nil {
-			oscapStageOptions.ProfileID = tailorConfig.Config.NewProfile
-			oscapStageOptions.Tailoring = tailorConfig.Filepath
-		}
-
-		directories, err := oscap.CreateRequiredDirectories(oscapConfig.Tailoring != nil)
-		if err != nil {
-			panic(err)
-		}
-
-		if len(directories) > 0 {
-			osc.Directories = append(osc.Directories, directories...)
-		}
+	var directories []*fsnode.Directory
```

Comment:
nitpick: consider renaming this to something like `oscapDirectories` to make it obvious what the variable is used for.